$NetBSD: patch-configure.ac,v 1.2 2026/07/19 11:17:41 adam Exp $

Enable COBOL on NetBSD amd64/arm64/riscv64.

Support Darwin/aarch64.
https://github.com/Homebrew/homebrew-core/blob/main/Patches/gcc/gcc-15.3.0.diff

--- configure.ac.orig	2026-06-12 06:09:05.615510417 +0000
+++ configure.ac
@@ -801,14 +801,14 @@ case ,${enable_languages}, in
     ;;
   *)
     case "${target}" in
-      aarch64-*-linux*|x86_64-*-linux*)
+      aarch64-*-linux*|x86_64-*-linux*|aarch64-*-netbsd*|x86_64-*-netbsd*|riscv64-*-netbsd*)
         ;;
       *-*-*)
         unsupported_languages="$unsupported_languages cobol"
         ;;
     esac
     case "${host}" in
-      aarch64-*-linux*|x86_64-*-linux*)
+      aarch64-*-linux*|x86_64-*-linux*|aarch64-*-netbsd*|x86_64-*-netbsd*|riscv64-*-netbsd*)
         ;;
       *-*-*)
         unsupported_languages="$unsupported_languages cobol"
@@ -819,6 +819,8 @@ esac

 # Disable libffi for some systems.
 case "${target}" in
+  aarch64*-*-darwin2*)
+    ;;
   powerpc-*-darwin*)
     ;;
   i[[3456789]]86-*-darwin*)
